dc-cudami-editor/src/components/IdentifiableSearch.jsx

```diff
@@ -12,13 +15,33 @@ const IdentifiableSearch = ({onChange, onSubmit, value}) => {
         onSubmit()
       }}
     >
-      <InputGroup>
-        <Input
-          onChange={(evt) => onChange(evt.target.value)}
-          placeholder={t('searchTerm')}
-          type="text"
-          value={value}
-        />
+      <InputGroup className="flex-nowrap">
+        <div className="position-relative">
+          <Input
+            className={classNames(
+              'pr-5',
+              isHighlighted && ['border', 'border-danger', 'border-width-3']
+            )}
+            onChange={(evt) => onChange(evt.target.value)}
+            placeholder={t('searchTerm')}
+            type="text"
+            value={value}
+          />
+          <Button
```

Comment:
Ich glaube, den `Button` könntest du besser in ein `InputGroupAddon` packen :thinking:

Reply:
Ne, dann siehts nicht so aus, wie ich es gerne hätte ;-) Der Button soll auf dem Input liegen. Daher auch `position-relative` und `position-absolute`